code/modules/flufftext/Hallucination.dm

```diff
@@ -1197,3 +1197,41 @@ GLOBAL_LIST_INIT(hallucinations_major, list(
 	H.preparePixelProjectile(target, start)
 	H.fire()
 	qdel(src)
+
+//Reality Dissociation Syndrome hallucinations only trigger in special cases and have no cost
+/datum/hallucination/rds
+	cost = 0
+
+/datum/hallucination/rds/fourth_wall/New(mob/living/carbon/C, forced = TRUE)
+	..()
+	to_chat(C, "<span class='userdanger extremelybig'>[pick("Leave the server" , "Close the game window")] [pick("immediately", "right now")].</span>")
+
+/datum/hallucination/rds/supermatter/New(mob/living/carbon/C, forced = TRUE)
+	..()
+	SEND_SOUND(C, 'sound/magic/charge.ogg')
+	to_chat(C, "<span class='boldannounce'>You feel reality distort for a moment...</span>")
```

Comment:
you could roll this hallucination inside /datum/hallucination/sounds, it already has the malf ai, and nuke op warning, as well as ratvar rising (a bit outated)